src/testing/deploy/upgrade_test.go

```diff
@@ -123,45 +156,61 @@ func TestUpgradeTrigger(t *testing.T) {
 			for i := 0; i < 11; i++ {
 				require.NoError(t, c.PutFile(dataCommit, "/hello", strings.NewReader("hello world")))
 			}
-			latestDataCI, err := c.InspectCommit(pfs.DefaultProjectName, dataRepo, "master", "")
 			require.NoError(t, err)
+			expectedCommitCount := getExpectedCommitCountFromVersion(from)
 			require.NoErrorWithinTRetry(t, 2*time.Minute, func() error {
-				ci, err := c.InspectCommit(pfs.DefaultProjectName, "TestTrigger2", "master", "")
+				commits, err := c.ListCommit(client.NewRepo(pfs.DefaultProjectName, "TestTrigger1"), nil, nil, 0)
```

Comment:
inspecting for an alias commit doesn't work on 2.8.x, so now we use the number of commits to verify all versions. we were already doing this in the post upgrade function.

Reply:
Also, when creating the pipeline you use the variable `pipeline1` but here you're using the hard-coded string.  Switch all instances of the string to the variable.

Reply:
okay, fixed the hard-coded part. let's read the commits with my log thingy and see what we get.

Reply:
https://pachyderm.atlassian.net/browse/PFS-223